code/datums/components/storage/storage.dm

```diff
@@ -848,10 +848,13 @@
 		return
 	A.add_fingerprint(user)
 	remove_from_storage(I, get_turf(user))
-	if(!user.put_in_hands(I))
-		to_chat(user, "<span class='notice'>You fumble for [I] and it falls on the floor.</span>")
+	INVOKE_ASYNC(src, .proc/quickdraw.quickdraw, user, I)
```

Comment:
I have no idea what `.proc/quickdraw.quickdraw` is, you should be able to just use `.proc/quickdraw`?

Reply:
it was a typo